nilearn/_utils/docs.py

```diff
@@ -1038,6 +1038,46 @@ def custom_function(vertices):
     The title displayed on the figure.
 """
 
+# transparency
+docdict["transparency"] = """
+transparency : :obj:`float` between 0 and 1, \
+                or a Niimg-Like object, \
+                or None, \
+                default = None
+    Value to be passed as alpha value to :func:`~matplotlib.pyplot.imshow`.
+    if ``None`` is passed, it will be set to 1.
+    If an image is passed, voxel-wise alpha blending will be applied,
+    by relying on the absolute value of ``transparency`` at each voxel.
+
+    .. versionadded:: 0.11.2
+"""
+
+# transparency
+docdict["transparency_range"] = """
+transparency_range : :obj:`tuple` or :obj:`list` of 2 real numbers, \
+                or None, \
+                default = None
+    When an image is passed to ``transparency``,
+    this determines the range of values in the image
+    to use for transparency (alpha blending).
+    For example with ``transparency_range = [1.96, 3]``,
+    any voxel / vertex (:math:`v_i`):
+
+    - with a value less than 1.96 would be fully transparent (alpha = 0),
+    - with a value greater then 3 than would be fully opaque (alpha = 1),
+    - with a value in the interval ``[1.96, 3.0]`` would have an alpha_i value
+      scaled linearly between 0 and 1 :
+      :math:`alpha_i = (v_i - 1.96) / (3.0 - 1.96)`.
+
```

Comment:
Just wondering: would it make sense to have a "two-tailed version" that would act symmetrically on  the negative and positive intervals ?